examples/intensity-recipe.yaml

```diff
@@ -1258,20 +1258,31 @@ ParticleList:
 
 Dynamics:
   J/psi:
-    Type: nonDynamic
+    Type: NonDynamic
+    FormFactor:
+      Type: BlattWeisskopf
+      MesonRadius: { Value: 1.0 }
```

Comment:
Can a meson radius have an uncertainty?

Reply:
yeah, it can be a fit parameter just like mass

Reply:
resolve it, if you are ok with it